ProjectLighthouse.Servers.Website/Pages/SlotSettingsPage.cshtml

```diff
@@ -52,6 +52,37 @@ function onSubmit(){
                         <input type="hidden" name="avatar" id="avatar-encoded">
                     </div>
                 </div>
+
+                <div class="ui divider"></div>
+                <div class="ui red label">
+                    <i class="lock icon"></i>
+                    Locked
```

Comment:
```suggestion
                    <label for="locked">
                        <i class="lock icon"></i>
                        Locked
                    </label>
```
By wrapping this in a label with a `for` attribute you can allow the checkbox to be clickable from almost anywhere in the box rather than the checkbox itself